resources/functionMap.php

```diff
@@ -5016,7 +5016,7 @@
 'Imagick::waveImage' => ['bool', 'amplitude'=>'float', 'length'=>'float'],
 'Imagick::whiteThresholdImage' => ['bool', 'threshold'=>'mixed'],
 'Imagick::writeImage' => ['bool', 'filename='=>'string'],
-'Imagick::writeImageFile' => ['bool', 'filehandle'=>'resource'],
+'Imagick::writeImageFile' => ['bool', 'filehandle'=>'resource', 'format'=>'string'],
```

Comment:
since it's 
![image](https://github.com/user-attachments/assets/f965cf84-6c3a-4c14-8b97-c75a2e88fca5)
should it be 
```suggestion
'Imagick::writeImageFile' => ['bool', 'filehandle'=>'resource', 'format'=>'?string'],
```
?

Reply:
@kingjia90 Yes, the parameter is nullable, and its default value is `null`:
https://github.com/Imagick/imagick/blob/28f27044e435a2b203e32675e942eb8de620ee58/Imagick.stub.php#L222

Additionally, the `Imagick::writeImagesFile` method has a `format` parameter, which we should also add there.